Pod/Server/UIViewController+SBTUITestTunnel.h

```diff
@@ -23,6 +23,7 @@
 #if ENABLE_UITUNNEL
 
 @import Foundation;
+@import UIKit;
```

Comment:
Given that UIKit imports Foundation we can have a single import here.

```suggestion
@import UIKit;
```